src/common/image.c

```diff
@@ -46,6 +46,43 @@
 #endif
 #include <glib/gstdio.h>
 
+int64_t max_image_position()
```

Comment:
if it's a function only used in this file then make it `static`.

Reply:
Done